src/lib/logical_query_plan/export_node.hpp

```diff
@@ -13,12 +15,14 @@ namespace hyrise {
  */
 class ExportNode : public EnableMakeForLQPNode<ExportNode>, public AbstractNonQueryNode {
  public:
-  ExportNode(const std::string& init_file_name, const FileType init_file_type);
+  ExportNode(const std::string& init_file_name, const FileType init_file_type,
+             const std::optional<EncodingType> init_file_encoding = std::nullopt);
```

Comment:
I don't think we need to maintain the encoding for `ExportNode`s as we won't implement the feature in the near future. I't basically dead code with the Assertion in SqlTranslator.